exercises/twelve-days/test/Tests.hs

```diff
@@ -0,0 +1,146 @@
+{-# LANGUAGE RecordWildCards #-}
+
+import Data.Foldable     (for_)
+import Test.Hspec        (Spec, describe, it, shouldBe)
+import Test.Hspec.Runner (configFastFail, defaultConfig, hspecWith)
+
+import TwelveDays (recite)
+
+main :: IO ()
+main = hspecWith defaultConfig {configFastFail = True} specs
+
+specs :: Spec
+specs = describe "responseFor" $ for_ cases test
+  where
+      test Case{..} = it description assertion
+        where
+          assertion = recite start stop `shouldBe` expected
+
+data Case = Case { description :: String
+                 , start       :: Int
+                 , stop        :: Int
+                 , expected    :: [String]
+                 }
+
+cases :: [Case]
+cases = [ Case { description = "first day a partridge in a pear tree"
+               , start = 1
+               , stop = 1
+               , expected = [ 
+                  "On the first day of Christmas my true love gave to me, a Partridge in a Pear Tree."
```

Comment:
I may like to change some alignments, two spaces in, so that the commas (were they to be present) align with the closing brace. not sure if there is a strict standard here but what I propose seems reasonable.